learning/basics/task1/AC-input_print.py

```diff
@@ -0,0 +1,18 @@
+x = input("Hello! What is your name? ")
+print("Hello " + x + "!, give me two numbers and i will multiply them.")
+y = 0
+z = 0
+while y == 0:
+    y_str = input("First -> ")
+    try:
+        y = int(y_str)
+    except:print("Sorry, numbers only :)")
+
```

Comment:
Clever, but what if I would like to multiply by 0? There will be infinite loop :) Consider initial `y`, and `z` values as `None`, then `while x == None:`